### DISCUSSION TITLE
`vim::MoveToNext` but with all the search options disabled?

### DISCUSSION THREAD

**Discussion by osa1** [zed-industries/zed#15837] [Feature Requests]
I'm looking for a command like `vim::MoveToNext`, but the command should either give me the options to disable case sensitivity and regex search options, or should come with the options disabled.

`vim::MoveToNext` comes hard-coded with those options: https://github.com/zed-industries/zed/blob/82db5dedfb40aa2dbc1320d0c29882c2015a553f/crates/vim/src/normal/search.rs#L268

Does such a command exist?

If not, would it make sense to update `vim::MoveToNext` (and `vim::MoveToPrev`) to allow setting these options?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
